### PR TITLE
Vendure provider

### PR DESCRIPTION
**Pull request by michaelbromley**
Hi!

Here's an updated PR for a [Vendure](https://www.vendure.io/) provider. Docs for testing it out are in the readme. Sorry about the messy branch history!

### CODE REVIEW

File: components/ui/LoadingDots/LoadingDots.tsx [hunk 1]
```diff
@@ -3,9 +3,6 @@ import s from './LoadingDots.module.css'
 const LoadingDots: React.FC = () => {
   return (
     <span className={s.root}>
-      <span />
-      <span />
```
Comment (okbel) [inline components/ui/LoadingDots/LoadingDots.tsx:7]: ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## Changelog
+
+- Select Variants Working
```
Comment (okbel) [inline CHANGELOG.md:3]: This is old!

**Comment by okbel** [comment]
Could you rebase with master? :D Thank you Michael!

**Comment by michaelbromley** [comment]
Ok, rebased and cleaned up the history!

**Comment by zohra-alt** [comment]
The integration of next.js commerce with vendure is still in development or this is the final version?

**Pull request merged**